sandbox/lua/modules/line_protocol.lua

```diff
@@ -0,0 +1,246 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
How about 'ts_line_protocol' since it's being used for different time series databases (and could be expanded to others that implement a 'line protocol' in the future)?